tdms/src/iterator.cpp

```diff
@@ -6150,47 +6149,6 @@ void initialiseDouble2DArray(double **inArray, int i_lim, int j_lim) {
     for (int i_var = 0; i_var < i_lim; i_var++) inArray[j_var][i_var] = 0.0;
 }
 
-//result gives field according to the exp(-iwt) convention

```

Comment:
Should this comment be preserved somewhere?
(Maybe [line 57 of the new](https://github.com/t-young31/ifdtd/pull/7/commits/ce7b7fdec9a8a4abf476db7ec7b15d1f410100e7#diff-f613571e3523d1c68d7a9b71bc905e297c0bb6646ef8ac4c0602f1d2461696eeR57))

Reply:
it's preserved [here](https://github.com/t-young31/ifdtd/blob/ce7b7fdec9a8a4abf476db7ec7b15d1f410100e7/tdms/include/field.h#L179)